pdm.lock

```diff
@@ -3,10 +3,9 @@
 
 [metadata]
 groups = ["default", "docs", "email", "linting", "memray", "mypy", "testing", "testing-extra"]
-cross_platform = true
-static_urls = false
-lock_version = "4.3"
-content_hash = "sha256:3bf8ccaefcdd7ac0c02606dbb435934025578ce34798b72ea97bd85e39ac6702"
+strategy = ["cross_platform"]
+lock_version = "4.4.1"
```

Comment:
@sydney-runkle you can probably run `pdm self update` locally, I think it will avoid `lock_version` conflicts in the future :)